doc/nrf/releases/release-notes-latest.rst

```diff
@@ -217,7 +217,7 @@ Trusted Firmware-M:
 MCUboot
 =======
 
-The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``c74c551ed6``, plus some |NCS| specific additions.
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``3f49b5abf3``, plus some |NCS| specific additions. |NCS| specific mcuboot glue code has been moved to :file:`ncs/nrf/modules/mcuboot`.
```

Comment:
```suggestion
The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``3f49b5abf3``, plus some |NCS| specific additions.

The code for integrating MCUboot into |NCS| is located in :file:`ncs/nrf/modules/mcuboot`.
```

Reply:
This must also be updated in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/mcuboot/readme-ncs.html @tejlmand

Reply:
yep, ready here: https://github.com/nrfconnect/sdk-nrf/pull/3833